Sources/CompareComplyV1/InsecureConnection.swift

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2018, 2020.
+ * (C) Copyright IBM Corp. 2018, 2021.
```

Comment:
we should be running the script to remove copyright only updates

Reply:
These copyright changes are actually correct as they should have been added in a prior release